packages/react-native-codegen/src/parsers/parsers-commons.js

```diff
@@ -100,10 +107,73 @@ function emitMixedTypeAnnotation(
   });
 }
 
+function getTypeScriptMemberType(maybeEnumDeclaration: $FlowFixMe) {
+  if (maybeEnumDeclaration.members[0].initializer) {
+    return maybeEnumDeclaration.members[0].initializer.type
+      .replace('NumericLiteral', 'NumberTypeAnnotation')
+      .replace('StringLiteral', 'StringTypeAnnotation');
+  }
+
+  return 'StringTypeAnnotation';
+}
+
+function getFlowMemberType(maybeEnumDeclaration: $FlowFixMe) {
+  return maybeEnumDeclaration.body.type
+    .replace('EnumNumberBody', 'NumberTypeAnnotation')
+    .replace('EnumStringBody', 'StringTypeAnnotation');
+}
```

Comment:
Can you create a function in the `parser.js` interface called: `getMaybeEnumMemberType(maybeEnumDeclaration: $FlowFixMe)` and move the `getTypeScriptMemberType` implementation in the `TypeScriptParser` and the `getFlowMemberType` implementation in the `FlowTypeParser`?

Reply:
Sure, done